src/UpdateTracker.js

```diff
@@ -41,14 +41,17 @@ export default class UpdateTracker {
   unsubscribe(...urls) {
     for (let url of urls) {
       url = url.replace(/#.*/, '');
-      if (url in subscribers)
+      if (url in subscribers) {
         subscribers[url].delete(this.subscriber);
+        delete subscribers[url];
+        delete webSockets[new URL(url).host];
```

Comment:
There is an issue here if there are multiple subscribed resources under the same domain. I'll change this to only delete if its the last resource with the domain.